scripts/datasets/process_nerfstudio_to_sdfstudio.py

```diff
@@ -17,15 +19,23 @@ def main():
 
     parser = argparse.ArgumentParser(description="preprocess scannet dataset to sdfstudio dataset")
 
-    parser.add_argument("--data", dest="input_path", help="path to scannet scene")
+    parser.add_argument("--data", dest="input_path", help="path to polycam/colmap data directory")
     parser.set_defaults(input_path="NONE")
 
-    parser.add_argument("--output-dir", dest="output_path", help="path to output")
+    parser.add_argument("--output-dir", dest="output_path", help="path to output directory")
     parser.set_defaults(output_path="NONE")
 
-    parser.add_argument("--type", dest="type", default="colmap", choices=["colmap", "polycam"])
+    parser.add_argument("--type", dest="type", required=True, choices=["colmap", "polycam"])
     parser.add_argument("--indoor", action="store_true")
 
+    parser.add_argument("--crop-mult", dest="crop_mult", type=int, default=1)
+    parser.add_argument("--mono-prior", action="store_true")
+    parser.add_argument("--omnidata_path", dest="omnidata_path", help="path to omnidata model")
+    parser.set_defaults(omnidata_path="/home/pablo/0Dev/repos/omnidata/omnidata_tools/torch/")
```

Comment:
Do you mind change the default value to "/home/yuzh/Projects/omnidata/omnidata_tools/torch/" so it's consistent with extract_monocular_cues.py